instrumentation/src/javatest/org/oppia/android/instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+"""
+This library contains the test suite source files of the instrumentation module.
+Note that:
+ - Each source file has its own kt_android_library target
+ - Target name of each kt_android_library as to be in small letters, separated by a underscore
+ and ends with "_lib"
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+# Common deps for each test suite.
+android_library(
+    name = "instrumentation_deps",
+    testonly = True,
+    exports = [
+        "//third_party:androidx_test_ext_junit",
+        "//third_party:androidx_test_runner",
+        "//third_party:androidx_test_uiautomator_uiautomator",
+        "//third_party:com_google_truth_truth",
+        "//third_party:junit_junit",
+    ],
+)
+
+kt_android_library(
+    name = "exploration_player_test_lib",
```

Comment:
Shouldn't there also be an instrumentation_test target defined? I know that it's not working with bazel test currently, but we should still set it up, I think.